Runtime/Interfaces/IBacktraceMetrics.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Backtrace.Unity.Model.Metrics;
+using System.Collections.Generic;
+
+namespace Backtrace.Unity.Interfaces
+{
+    public interface IBacktraceMetrics
+    {
+        /// <summary>
+        /// This list contains the unique events which will be sent whenever Send is triggered (manually or automatically). 
+        /// By default, this list contains the guid attribute. This list is mutable, which means the default can be removed
```

Comment:
This list is mutable but what does it mean to mutate this list since we can only `get` it?

Reply:
List has.

Reply:
I agree. But why do we even need to `get` this then? Does the user care about using this list?

Reply:
In special case if user has to manipulate list - do any operations on them like for example removing them

Reply:
But this list is just a copy right? I think changing this list won't really do anything

Reply:
Its not a copy. Its a reference. You can manipulate objects three. However we'rr changing unique events and this wont be available

Reply:
Is there some kind of c# language trick here? It looks like we are just using the c# get/set syntax here which returns a value not a reference? I just want to understand, I know it doesn't really matter because we're changing this anyway

Reply:
Okay nevermind, I did some more reading and I understand now. You can ignore this lol thank you!